web/src/components/HiddenLLMHelper.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { useAllDocsData } from '@docusaurus/plugin-content-docs/client'
+
+export function HiddenLLMHelper() {
+  const allDocs = useAllDocsData()
+  const allLinks = Object.values(allDocs)
+    .flatMap((docData) => docData.versions[0].docs)
+    .filter((doc) => {
+      const excludedDocs = [
+        'vision',
+        'writingguide',
+        'telemetry',
+        'contact',
+        'contributing',
+        'migration-guides',
```

Comment:
I'm assuming that these aren't relevant for 99% of developers questions when working with Wasp via an LLM. I also read that having too many vectorized data points makes RAG less efficient, so I think it's best to keep it focused on the development docs. Migration guides may be useful, but I had issues where the LLM gave outdated version code as suggestions, so I want to avoid that.

Reply:
Makes sense! Maybe add that as a comment as I don't think it is instantly obvious.